wazo_dird/plugins/csv_backend/plugin.py

```diff
@@ -105,6 +114,14 @@ def _load_file(self):
         except IOError:
             logger.exception('Could not load CSV file content')
 
+    def _check_for_csv_file_update(self, filename):
+        fname = pathlib.Path(filename)
+        tmp_csv_file_last_modification_date = fname.stat().st_mtime
+
+        if self._csv_last_modification_time != tmp_csv_file_last_modification_date:
+            self._load_file()
+            self._csv_last_modification_time = tmp_csv_file_last_modification_date
```

Comment:
During the `load` of the plugin `_load_file` is called (line 50) and the `_csv_last_modification_time` is not updated. I think this line should be moved to the `_load_file` function. And maybe move all of this logic to `_load_file` and call it from each functions.